lib/smart_answer_flows/coronavirus-business-support.rb

```diff
@@ -0,0 +1,133 @@
+module SmartAnswer
+  class CoronavirusBusinessSupportFlow < Flow
+    def define
+      start_page_content_id "89edffd2-3046-40bd-810c-cc1a13c05b6a"
+      flow_content_id "1f589327-a6b3-4b5c-aea0-7a2752e2eddf"
```

Comment:
Curious where do these come from?

Reply:
Turns out we need to make our own, as per https://github.com/alphagov/smart-answers/blob/master/doc/smart-answer-flow-development/content-ids.md

Reply:
@theseanything Probably here https://github.com/alphagov/smart-answers/blob/master/doc/smart-answer-flow-development/content-ids.md

Reply:
@benthorner snap